lib/twemoji.rb

```diff
@@ -106,13 +105,12 @@ def self.parse(text, asset_root: "https://twemoji.maxcdn.com/",
                        file_ext:   ".png",
                        image_size: "16x16",
                        class_name: "emoji",
-                       img_attr: nil)
+                       img_attr: {})
```

Comment:
ok so this old api is intended to pass all attributes in one string, now we should rename to `img_attrs`.